backend/tests/_bootstrap.php

```diff
@@ -6,4 +6,4 @@
 require_once(YII_APP_BASE_PATH . '/vendor/autoload.php');
 require_once(YII_APP_BASE_PATH . '/vendor/yiisoft/yii2/Yii.php');
 require_once(YII_APP_BASE_PATH . '/common/config/bootstrap.php');
-require_once(YII_APP_BASE_PATH . '/backend/config/bootstrap.php');
\ No newline at end of file
+require_once(__DIR__ . '/../config/bootstrap.php');
```

Comment:
From commit message:
Made config files less module-aware by removing explicit references to 'backend' or 'frontend' directories

I find it helpful when adding a new "module" because there is one file less that requires adaption after copying either of the existing modules.